packages/amazonq/src/lsp/client.ts

```diff
@@ -131,7 +127,7 @@ export async function startLanguageServer(
                     name: env.appName,
                     version: version,
                     extension: {
-                        name: 'AmazonQ-For-VSCode',
+                        name: clientName,
```

Comment:
I think this might have to stay as `'AmazonQ-For-VSCode'`, it has some internal meaning in flare that I haven't figured out yet